src/components/ContentHighlights/HighlightStepper/ContentHighlightStepper.jsx

```diff
@@ -69,7 +69,10 @@ const ContentHighlightStepper = ({ enterpriseId }) => {
       const newHighlightSet = {
         title: highlightTitle,
         isPublished: true,
-        // TODO: pass along the selected content keys!
+        // TODO: This is a temporary solution to get the content keys from the
+        // selected row ids based on content_type (uuid, key etc.).
```

Comment:
FWIW, i think we can remove the `TODO` comment here for now. I don't imagine we'll change this anytime soon.